core-services/secret-mirroring/_mapping.yaml

```diff
@@ -432,29 +432,23 @@ secrets:
     namespace: baremetal
     name: packet-project-id
   to:
-    namespace: cluster-secrets-metal
+    namespace: cluster-secrets-packet
```

Comment:
`cluster-secrets-packet` is the [name](https://github.com/openshift/release/blob/7d13ec291e76c0f85da5d66fd20751add97883a9/core-services/ci-secret-bootstrap/_config.yaml#L1382) of the secret, the [namespace](https://github.com/openshift/release/blob/7d13ec291e76c0f85da5d66fd20751add97883a9/core-services/ci-secret-bootstrap/_config.yaml#L1383) is `ci`